torch/autograd/gradcheck.py

```diff
@@ -214,6 +224,9 @@ def gradcheck(
 
     The check between numerical and analytical gradients uses :func:`~torch.allclose`.
 
+    For functions with complex output, the gradients are computed using :attr:`grad_output`
```

Comment:
I'm not sure what that means.
The matrix that you compare is the matrix containing all the conjugate wirtinger derivatives computed under the assumption that the overall function is C -> R no?
Also to keep this short, you can simply point to the autograd Note on complex if people want to know the definition of these terms.

Reply:
updated the description